sdk/src/test/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessorTest.java

```diff
@@ -230,7 +230,7 @@ public void serviceHandlerThrowsException() {
     assertThat(exported).containsExactly(span2.toSpanData());
   }
 
-  @Test
+  @Test(timeout = 5000)
```

Comment:
I would expect this to be smaller than 1000 to prove that the timeout occur

Reply:
yeah, this needs to be shorter than the timeout or we won't be testing anything.

Reply:
We test that an InterruptedException is thrown in the exporter, otherwise the `await` will never return.

Reply:
oh you're totally right. the no-timeout version on the latch makes this work. sorry for the brain-fart.